mne_bids/tests/test_path.py

```diff
@@ -528,14 +527,26 @@ def test_get_matched_basenames(return_bids_test_dir):
     """Test retrieval of matching basenames."""
     bids_root = return_bids_test_dir
 
-    paths = get_matched_basenames(bids_root=bids_root)
+    bids_path_01 = BIDSPath(root=bids_root)
+    paths = bids_path_01.match()
     assert len(paths) == 7
     assert all('sub-01_ses-01' in p.basename for p in paths)
     assert all([p.root == bids_root for p in paths])
 
-    paths = get_matched_basenames(bids_root=bids_root, run='01')
+    bids_path_01 = BIDSPath(root=bids_root, run='01')
+    paths = bids_path_01.match()
     assert len(paths) == 3
     assert paths[0].basename == 'sub-01_ses-01_task-testing_run-01_channels'
 
-    paths = get_matched_basenames(bids_root=bids_root, subject='unknown')
+    bids_path_01 = BIDSPath(root=bids_root, subject='unknown')
+    paths = bids_path_01.match()
     assert len(paths) == 0
+
+    # now test it with BIDSPath
```

Comment:
I don't understand this comment, we've been using BIDSPath all along in this test up until here, now?